### PR TITLE
Add error details to error banner in migration cutover dialog

### PR DESCRIPTION
**Pull request by goyal-anjali**
This PR fixes 

- Info icon for dark mode was set to non-existing icon
- Added migration error details to the Error banner in the migration cutover dialog - additional error details added to the banner detail section from the 'Copy Migration Details" button

### CODE REVIEW

**Comment by goyal-anjali** [comment]
Before -
MicrosoftTeams-image (4)
After -
image

**Comment by goyal-anjali** [comment]
What is the solution for PR Validation check failing?

**Comment by Charles-Gagnon** [comment]
You have two options when a check fails : 

1. Fix the issue
2. Re-run the validation

Since the error appears to be a generic timeout issue with Github it is likely not related to your changes so you should just try re-running the check. If it keeps failing then reach out to the team and we can help diagnose.

File: extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialog.ts [hunk 1]
```diff
@@ -435,6 +435,19 @@ export class MigrationCutoverDialog {
 		}
 	}
 
+	private getMigrationDetails(): string {
+		if (this._model.migrationOpStatus) {
```
Comment (aasimkhan30) [inline extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialog.ts:439]: Can you also use this method in the place where you got the code from?

https://github.com/microsoft/azuredatastudio/blob/e75572f5ca7e0b221141841e208679ee701ab455/extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialog.ts#L308-L308

**Review by aasimkhan30 — CHANGES_REQUESTED** [review]

File: extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialog.ts (hunk 1 above)
Comment (goyal-anjali) [reply extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialog.ts:432]: Done, Thanks!

**Review by brian-harris — APPROVED** [review]

**Pull request merged**